web-console/src/utils/ingestion-spec.tsx

```diff
@@ -1206,6 +1206,30 @@ export function getIoConfigFormFields(ingestionComboType: IngestionComboType): F
             </>
           ),
         },
+        {
+          name: 'inputSource.properties.accessKeyId',
+          label: 'Access key ID',
+          type: 'string',
+          placeholder: '(optional)',
+          info: (
+            <>
+              <p>S3 access key for this S3 bucket.</p>
+              <p>Setting this will override the default configuration provided in the config.</p>
```

Comment:
provided in which config? 

Also default configuration -> access key used for deep storage?

Same comment on line 1229

Reply:
I don't think it has to be the deep storage one (as in you might now be using s3 for deepstorage) if I understand correctly it is just the key from the config that will be used if this is not set.

Reply:
The defaults are the keys provided from the S3 credentials providers chain. See more information at : https://druid.apache.org/docs/latest/development/extensions-core/s3.html#s3-authentication-methods
Which may or may not be from configs (i.e. If Druid config file and Custom properties file are not set but the env variable are, then the credentials providers chain will use the env variable)

Reply:
Maybe we can say:
Setting this will override the value provided from the S3 credentials providers chain